api/handlers.go

```diff
@@ -182,6 +233,38 @@ func (api *API) userStatsGET(w http.ResponseWriter, req *http.Request, _ httprou
 	api.WriteJSON(w, us)
 }
 
+// userPOST creates a new user.
+func (api *API) userPOST(w http.ResponseWriter, req *http.Request, _ httprouter.Params) {
+	err := req.ParseForm()
+	if err != nil {
+		api.WriteError(w, err, http.StatusInternalServerError)
+		return
+	}
+	email := req.PostFormValue("email")
+	// TODO Validation
```

Comment:
I'd add this right away. Just need to do `address, err := mail.Parse(email)`
It's in the stdlib.

Reply:
Thanks, this is what I was looking for. :)